### PR TITLE
[INFRA,TEST] Include new tests for format_help.hpp to improve code coverage

### PR DESCRIPTION
**Pull request by joshuak94**
More commits incoming with format_html.hpp and format_man.hpp

Pull request isn't completed yet, will assign reviewer when I'm ready!

### CODE REVIEW

File: test/unit/argument_parser/detail/format_help_test.cpp [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+// ARE DISCLAIMED. IN NO EVENT SHALL KNUT REINERT OR THE FU BERLIN BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+// LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+// OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+// DAMAGE.
+//
+// ============================================================================
```
Comment (h-2) [inline test/unit/argument_parser/detail/format_help_test.cpp]: short copyright

File: test/unit/argument_parser/detail/format_html_test.cpp [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+// ARE DISCLAIMED. IN NO EVENT SHALL KNUT REINERT OR THE FU BERLIN BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+// LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+// OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+// DAMAGE.
+//
+// ============================================================================
```
Comment (h-2) [inline test/unit/argument_parser/detail/format_html_test.cpp]: here as well

**Comment by h-2** [comment]
can you make the small changes so we can merge this before the meeting today?

File: test/unit/argument_parser/detail/format_help_test.cpp [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+
+#include <seqan3/argument_parser/all.hpp>
+#include <seqan3/argument_parser/detail/format_help.hpp>
+#include <seqan3/io/stream/parse_condition.hpp>
+
+using namespace seqan3;
+
+TEST(help_add_test, add_option)
+{
+<<<<<<< HEAD
```
Comment (smehringer) [inline test/unit/argument_parser/detail/format_help_test.cpp]: Looks like you did not resolve a rebase?

**Pull request merged**